lib/super_diff/rspec/monkey_patches.rb

```diff
@@ -782,7 +782,8 @@ def self.prepended(base)
       end
 
       def match_array(items)
-        BuiltIn::MatchArray.new(items.is_a?(String) ? [items] : items)
+        items = *items
+        BuiltIn::MatchArray.new(items)
```

Comment:
Shouldn't this be?
```suggestion
        BuiltIn::MatchArray.new(items.is_a?(Array) ? items : [items] )
```

Reply:
The original implementation is splatting the given argument, so I believe it would be more strict to do the same conversion.
https://github.com/rspec/rspec-expectations/blob/040d1b0bba83031c9ae432fd3a4462fdd1cc74e7/lib/rspec/matchers.rb#L715-L717

Reply:
But, the argument of `match_array` can be non-array as well. https://rubydoc.info/github/rspec/rspec-expectations/RSpec%2FMatchers:match_array

This is why the tests are breaking in my project.

Reply:
👍🏼

Reply:
Good call!